### DISCUSSION TITLE
[Feature] Pin albums so that they appear first in the list in Sharing and Albums views, as a per user setting (web and mobile)

### DISCUSSION THREAD

**Discussion by jon-f-novabackup** [immich-app/immich#2321] [Feature Request]
Pin albums so that they appear first in the Sharing view, Albums view, and Shared albums list seen when adding assets via the Photos timeline view. The pin function would be enabled inside each album as an option (next to perhaps the other feature request to be able to sort assets inside an album, not just by date oldest to newest), and of course add a small pin icon next to the album name when viewing the album and when listing the album, if it is pinned. The order in which the pinned albums would always be first, but even the pinned albums (optimally) could be made to abide by each user's existing sort order preference. This request would be for both web and mobile. The reason being with items that are pinned are easier to find, if the user prefers it and needs that. I have 50 total albums but I only add assets mainly to the first few albums that were created in Immich, which if I'm in Photos timeline view and I want to add a single asset to one of those 3 shared album from that view, I have to expand the Shared albums list and then take 30 seconds for me to scroll down 50 albums just to get to the oldest 3 shared albums that were created to be able to add assets to them. The later problem would also be helped tremendously if feature request #2319 (for Shared albums view / list) and #1689 (for Albums view) were implemented, that deal with adding the ability to sort the albums list by most recently added images listed first. I searched for the word pin and pinned but didn't see a feature request for this.

**Comment by ztec** [comment]
I too would like a similar feature. 

Also, it would be nice to be able to decide to list them in a different manner than other albums like 
 - Only the cover/icon
 - Only the title
 - Small or Big (as big as other albums)

My usecase: 
Thanks to the Album synchronization feature, I now have some album that are perpetual. They are not here to gather photo related to a topic but rather a function. For example, I have one album with all my rush dumped. Another with some images processed but not yet put in dedicated albums. Another one with all photos sent/received by messages.  etc...

Sorting can be a solutions, but I would prefer to keep the sorting by date, and still have those albums on top, even if nothing happen to them for a long time.

**Comment by stephenmoore33** [comment]
Not sure if this should be a separate request, but IN ADDITION to pinning, the ability to "hide", or manually sort albums via the Immich app would be neat as well. 

I have 100s of albums, and would like to hide 70% of them, so I can de-clutter the screen.
